src/include/simeng/kernel/LinuxProcess.hh

```diff
@@ -100,6 +105,9 @@ class LinuxProcess {
   /** The process image. */
   char* processImage_;
 
+  /** The process image as a vector */
```

Comment:
Minor preference to end header file function comments with a full stop

Reply:
Done.